README.md

```diff
@@ -64,34 +64,89 @@ data
 ├── metric_datasets
 │   ├── dogs
 │   ├── shoes
+│   ├── celebrities
+│   ├── logos
 ├── models
 │   ├── dogs
 │   └── shoes
+│   └── celebrities
+│   └── logos
 └── qdrant_storage
     ├── aliases
     ├── collections
     │   ├── dogs
     │   └── shoes
+    │   └── celebrities
+    │   └── logos
     └── collections_meta_wal
 interactive
 metrics
 notebooks
 scripts
 
 ```
-This structure is split as follows:
+The structure of the `data` folder is split as follows:
   * `metric_datasets` - split into folders corresponding with data categories, each containing raw pictures that were used for model training and are being pulled as a result of visual search.
   * `models` - split into folders corresponding with data categories, each containing pretrained deep learning models,
   * `qdrant_storage` - storage for vector search engine (Qdrant), each data category has its own collection.
 
-### Local
+### Local - Manual
+
+Installation using the terminal window:
+* Install ***git***, ***docker*** packages.
+* `cd` to your target directory.
+* Clone [repository](https://github.com/stxnext/visual-similarity-search) (preferably use SSH cloning).
+* Download `data.zip` file from the [Google Drive](https://drive.google.com/file/d/1ru2FHcV8xBDqX9Q_zgAkJYDlYWlszmOD/view?usp=sharing) and unpack it to the repository so that the folder structure above is retained.
+* Install Python version 3.10 and ***pip***, ***pipenv*** libraries.
+```
+sudo apt-get install python3.10 
+curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
+python3.10 -m pip install pipenv  
+```
+* Set up local environment and run shell:
+```
+python3.10 -m pipenv --python 3.10
```

Comment:
Do you really need pipenv here? I think that poetry has compatibility with pyenv.

Reply:
Sometimes referenced poetry installation does not use the proper Python version defined in `.toml` files. Using shell takes care of that issue.